packages/create-flex-plugin/templates/craco.config.js

```diff
@@ -0,0 +1,10 @@
+const config = require('craco-config-flex-plugin');
+
+module.exports = {
+     ...config,
+     plugins: [
+          // You can customize the configuration by defining your own plugins.
```

Comment:
Defining your own plugins for your plugins?  This is pretty confusing at first glance, especially as it is one of only 4 files in the root of the project.

Reply:
just changing some docs/comments should do the trick

Reply:
😛 What about

```
You can customize app configuration (such as webpack, devServer, linter, etc) by creating a craco plugin. See https://github.com/sharegate/craco/tree/master/packages/craco#develop-a-plugin for more detail.
```